### ISSUE TITLE
New angular templates do not support (some) bootstrap intellisense

### ISSUE THREAD

**Issue by kakins** [aspnet/templating#364]
I just created a new project from the angular template using `dotnet new`.  I noticed now in Visual Studio I do not see some pretty common Bootstrap classes when I type.

![image](https://user-images.githubusercontent.com/14915247/37493924-af2a58ce-2874-11e8-9c43-384b99fa644b.png)

**Comment by Eilon** [comment]
@kakins for this type of Intellisense issue please use Visual Studio's built-in "Send Feedback" feature (located in the upper-right of VS's window:

![image](https://user-images.githubusercontent.com/202643/37498949-56852b22-287e-11e8-8f38-f6dee5339199.png)

That will ensure that the feedback goes to the VS team that works on that feature. Thanks!

**Comment by kakins** [comment]
@Eilon Ok I can do that.  I just found it strange that it **only** happens when I switch to the new Angular CLI-based templates.  The previous templates installed with Visual Studio have the bootstrap intellisense support.  Are we sure this is not an issue with the template project itself?

**Comment by Eilon** [comment]
@kakins understood, but one way or another as far as I know that's where to report almost all Intellisense issues for CSS/JS (there might be a few exceptions to that, but I don't think this is one of them).

**Comment by kakins** [comment]
@Eilon Will do, thanks!

**Comment by kakins** [comment]
If anyone is having this issue, I'm not sure if it's an issue with the templates _per se_, however intellisense did disappear when I switched to using the new templates.  I found a similar issue here:

https://irsali.github.io/others/2017/10/06/Visual-Studio-Ensuring-Intellisense.html

I'm really not sure what to do in this situation.  I don't want to add bootstrap to my project as the above post suggests just to get intellisense back.
